views/index.erb

```diff
@@ -0,0 +1,152 @@
+<style>
```

Comment:
Keeping styling separate instead of using inline CSS really helps in terms of readability, efficient too